include/core/shape.h

```diff
@@ -378,6 +378,178 @@ class Sphere : public Shape<dim>
 #endif
 };
 
+/**
+ * @class This class defines superquadric shapes. Their signed distance function
+ * is:
+ * |\frac{x}{a}|^r  + |\frac{y}{b}|^s + |\frac{z}{c}|^t - 1 = 0
```

Comment:
```suggestion
 * :math:`\left|\frac{x}{a}\right|^r + \left|\frac{y}{b}\right|^s + \left|\frac{z}{c}\right|^t - 1 = 0`
```

Slightly more aesthetic